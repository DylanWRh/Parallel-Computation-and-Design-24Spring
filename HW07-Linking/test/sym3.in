2 -2 -13 
12 -3 
-18 
