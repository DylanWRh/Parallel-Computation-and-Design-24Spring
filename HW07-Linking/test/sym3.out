Eigenvalues:
-18.5044
-16.0491
16.5534

Eigenvalues
0.31244 0.945264 -0.0941097 
-0.0552875 0.116996 0.991592 
0.948327 -0.30461 0.0888157 
